Taskfile.yml

```diff
@@ -30,15 +30,13 @@ tasks:
 
   deps:fe:
     desc: Installs npm requirements for front end from package.json
-    dir: public
+    dir: web/resources
     cmds:
-      - git submodule update --init --recursive
       - npm i -g less pug-cli
```

Comment:
is it possible to add these tools to the devDependencies of the package.json so we don't need to install them here?

Reply:
This global packages, we can't put these to package.json

Reply:
do they have to be globally installed though? is there a good reason for this? as really they are project level development dependencies

Reply:
https://github.com/pugjs/pug-cli - I'm not found examples of using it locally. 
I tested locally, it works fine, but not documented and looks like hack for me:
```node node_modules/pug-cli resources/pug --out public/html```